### ISSUE TITLE
[Bug]: recycle_bin directory setting not being respected (regression? v4.1.9)

### ISSUE THREAD

**Issue by SawkeeReemo** [StuffAnThings/qbit_manage#651]
### Describe the Bug

The recycle_bin directory setting in config seems to be getting ignored on this latest update [v4.1.9]

My directory structure:
```
video \
      downloads \
      orphaned \
      recycle \
               downloads \
               torrents \
               torrents_json \
```

qbm config:
`root_dir: /video`
`recycle_bin: /video/recycle`
`orphaned_dir: /video/orphaned`

The volume is mount the same in both the qbit and qbm containers: `${DOWNLOADS}:/video`

The orphaned files setting still works as expected.  But for recycles I'm noticing that qbm is creating a new directory `/video/downloads/recycle`[etc] instead of using the directory I've told it to. This used to work fine until couple days ago with this new release, but now it's suddenly putting things in this new folder. Did something change there? Regression?

### Config

```yaml
directory:
  cross_seed: /cross-seed
  root_dir: /video
  remote_dir: /video
  recycle_bin: /video/recycle
  torrents_dir: /qbtmisc_config/qBittorrent/BT_backup
  orphaned_dir: /video/orphaned
```

### Logs

https://gist.github.com/SawkeeReemo/2c286c70013b3c57e4c9a60afaf1f524

### Installation

Docker

### Version Number

4.1.9

### What branch are you on?

master

**Comment by bakerboy448** [comment]
not a bug; has not changed in years

https://github.com/StuffAnThings/qbit_manage/blame/66aec5b0ecccdd271f0021035d1fd04166dcb420/modules/qbittorrent.py#L430

recycle bin is the base;  torrents and json etc get their own folders

**Comment by bakerboy448** [comment]
log `/video/downloads/recycle/torrents` vs config `/video/recycle`

**Comment by SawkeeReemo** [comment]
Right… which is the problem. This is a bug. I’ve told it where to go, it’s always put it there, now with this latest release it’s putting it somewhere else. Nothing has changed other than this release update.

**Comment by bakerboy448** [comment]
> That’s the reason. The split by category will create a new recycle_bin folder under each category

**Comment by SawkeeReemo** [comment]
Yup. Got it. Not a bug, just not explained well in the wiki. This could be thought of more as “split by directory” than category. Meaning, for anyone reading, if your categories have different directories (mine all point to the same directory), then when you turn this option on, it will put a recycle folder inside each of those directories. It will not split up the recycle bin directory into subfolder named after your categories, which is how I understood the wiki to mean. 

Also, my own mistake, I forgot that I turned that option on the other day while I was doing 10,000 other things. So something DID change in my config, I just forgot I did it.
